moto/ses/responses.py

```diff
@@ -507,6 +524,57 @@ def describe_receipt_rule(self):
   </ResponseMetadata>
 </CreateReceiptRuleResponse>"""
 
+DESCRIBE_RECEIPT_RULE_SET = """<DescribeReceiptRuleSetResponse xmlns="http://ses.amazonaws.com/doc/2010-12-01/">
+  <DescribeReceiptRuleSetResult>
+    <Rules>
+      {% for rule in rule_set %}
+      <member>
+        <Recipients>
+          {% for recipient in rule["recipients"] %}
+          <member>{{recipient}}</member>
+          {% endfor %}
+        </Recipients>
+        <Name>{{rule["name"]}}</Name>
+        <Actions>
+          {% for action in rule["actions"] %}
+          <member>
+            {% if action["_s3_action"] %}
+            <S3Action>
+              <BucketName>{{action["_s3_action"]["_bucket_name"]}}</BucketName>
+              <KmsKeyArn>{{action["_s3_action"]["_kms_key_arn"]}}</KmsKeyArn>
+              <ObjectKeyPrefix>{{action["_s3_action"]["_object_key_prefix"]}}</ObjectKeyPrefix>
+              <TopicArn>{{action["_s3_action"]["_topic_arn"]}}</TopicArn>
+            </S3Action>
+            {% endif %}
+            {% if action["_bounce_action"] %}
+            <BounceAction>
+              <TopicArn>{{action["_bounce_action"]["_topic_arn"]}}</TopicArn>
+              <SmtpReplyCode>{{action["_bounce_action"]["_smtp_reply_code"]}}</SmtpReplyCode>
+              <StatusCode>{{action["_bounce_action"]["_status_code"]}}</StatusCode>
+              <Message>{{action["_bounce_action"]["_message"]}}</Message>
+              <Sender>{{action["_bounce_action"]["_sender"]}}</Sender>
+            </BounceAction>
+            {% endif %}
+          </member>
+          {% endfor %}
+        </Actions>
+        <TlsPolicy>{{rule["tls_policy"]}}</TlsPolicy>
+        <ScanEnabled>{{rule["scan_enabled"]}}</ScanEnabled>
+        <Enabled>{{rule["enabled"]}}</Enabled>
+      </member>
+      {% endfor %}
+    </Rules>
+    <Metadata>
+      <Name>{{rule_set_name}}</Name>
+      <CreatedTimestamp>2021-10-31</CreatedTimestamp>
```

Comment:
I'm not sure how to populate this value here.

We'd probably have to update the `[create_receipt_rule_set](https://github.com/spulec/moto/blob/1d21cec0004afae584e9c429238d402b888ae78c/moto/ses/models.py#L415)` method to save the datetime when the ruleset is created, but I'm not sure where we'd add the created datetime since the current data structure is just a dict with the rule set name as the key to a list of rules. i.e.

```
{
   "testRuleSet":[
      {
         "name":"testRule",
         "enabled":"true",
         ...,
      }
   ]
}
```

Is there any preference in how we implement this?

Reply:
Ideally the rule sets are stored as they're own class (class FakeRuleSet(BaseModel)), to be in-line with other data stores.

If this PR is ready, I'll merge it for now then - having proper metadata can be part of a next PR 👍